RELEASE_NOTES.md

```diff
@@ -7,6 +7,8 @@ A new header is inserted each time a *tag* is created.
 
 - engine: Attachments of custom RendereTargets are not systematically discarded
 - engine: New API to duplicate a `MaterialInstance`
+- engine: update SPIRV-Cross to the latest (sha: ff6189072) which gives us access to 
```

Comment:
I'm not sure if this needs to be in the public release notes, but up to you. Also, the rc/1.10.1 branch has been cut, so this should go under the new 1.10.2 header.